internal/cli/atlas/deployments/options/deployment_opts.go

```diff
@@ -44,13 +44,13 @@ type DeploymentOpts struct {
 	DeploymentType string
 	MdbVersion     string
 	Port           int
-	podmanClient   podman.Client
+	PodmanClient   podman.Client
```

Comment:
why exporting it?

Reply:
it's used in deployments/search/indexes/create PreRunE (init and initStore)